test-apps/appui-test-app/appui-test-providers/src/ui/providers/PopoutWindowsStageUiItemsProvider.tsx

```diff
@@ -0,0 +1,95 @@
+ * display to allow user to play the animation.
+ */
+export class PopoutWindowsStageUiItemsProvider implements UiItemsProvider {
+  public static providerId = "appui-test-providers:popout-windows-stage-items-provider";
+  public readonly id = PopoutWindowsStageUiItemsProvider.providerId;
+
+  constructor(localizationNamespace: string) {
+    OpenPopoutViewTool.register(localizationNamespace);
+    OpenPopoutDialogTool.register(localizationNamespace);
+  }
+
+  public static register(localizationNamespace: string) {
+    UiItemsManager.register(new PopoutWindowsStageUiItemsProvider(localizationNamespace), { stageIds: [PopoutWindowsFrontstage.stageId] });
+  }
+
+  public static unregister() {
+    UiItemsManager.unregister(PopoutWindowsStageUiItemsProvider.providerId);
+  }
+
+  public provideWidgets(_stageId: string, stageUsage: string, location: StagePanelLocation, section?: StagePanelSection): ReadonlyArray<Widget> {
+    const widgets: Widget[] = [];
+    if (stageUsage === StageUsage.General && location === StagePanelLocation.Left && section === StagePanelSection.Start) {
+      widgets.push({
+        id: "appui-test-providers:ViewAttributesWidget",
+        label: "View Attributes",
+        icon: "icon-window-settings",
+        defaultState: WidgetState.Open,
+        canFloat: {
+          containerId: "appui-test-providers:ViewAttributesWidget",
+        },
+        content: <ViewAttributesWidgetComponent />,
+        canPopout: true,
+        allowedPanels: [StagePanelLocation.Left, StagePanelLocation.Right],
+      });
+    }
+    return widgets;
+  }
+
+  public provideToolbarItems(stageId: string, _stageUsage: string, toolbarUsage: ToolbarUsage, toolbarOrientation: ToolbarOrientation): ToolbarItem[] {
+    const allowedStages = [PopoutWindowsFrontstage.stageId];
+    if (allowedStages.includes(stageId)) {
+      if (toolbarUsage === ToolbarUsage.ContentManipulation && toolbarOrientation === ToolbarOrientation.Horizontal) {
+        return [
+          OpenPopoutViewTool.getActionButtonDef(20, 3000),
+          OpenPopoutDialogTool.getActionButtonDef(25, 3000),
+        ];
+      } else if (toolbarUsage === ToolbarUsage.ViewNavigation && toolbarOrientation === ToolbarOrientation.Vertical) {
+        return [
+          getCustomViewSelectorPopupItem(20, 3000),
+        ];
+      }
+    }
+    return [];
+  }
+
+  public provideStatusBarItems(_stageId: string, stageUsage: string): StatusBarItem[] {
+    const statusBarItems: StatusBarItem[] = [];
+    if (stageUsage === StageUsage.General) {
+
+      statusBarItems.push(
+        StatusBarItemUtilities.createCustomItem("DisplayStyle", StatusBarSection.Center, 400, <DisplayStyleField />),
+      );
+    }
+    return statusBarItems;
+  }
+
+  public provideBackstageItems(): BackstageItem[] {
+    const label = AppUiTestProviders.translate("backstage.PopoutWindowsFrontstageLabel");
```

Comment:
Is it normal that we are not using the `localizationNamespace` here ?

Reply:
It's in AppUiTestProviders.translate().